torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -101,6 +101,73 @@ class _X86InductorQuantizationAnnotation(QuantizationAnnotation):
 QUANT_ANNOTATION_KEY = "quantization_annotation"
 
 
+def _skip_annotate(
+    nodes: List[Node], filter_fn: Optional[Callable[[Node], bool]] = None
+):
+    # 1) Skip annotate if any node is already annotated
+    if _is_any_annotated(nodes):
+        return True
+
+    # 2) Not skip annotate if a) filter_fn is provided and b) any node passed the filter
+    if filter_fn and any(filter_fn(node) for node in nodes):
+        return False
+
+    return True
+
+
+def _get_operator_type_filter(operator_type: Callable, module_name_list):
+    """Get the operator_type_filter function for a given operator type and module name list.
+
+    The filter accept a node and checks if 1) the node has certain operator type,
+    and 2) the node does not marked by `set_module_name_qconfig`.
+
+    For example:
+        # linear_op.target if torch.ops.aten.linear.default
+        # linear_op is traced from `fc3`
+        node: linear_op = call_function[...](...)
+
+
+    >> operator_type_filter = _get_operator_type_filter(torch.ops.aten.linear.default, ["fc1", "fc2"])
+    >> print(operator_type_filter(node))
+    True  # the node's target is `torch.ops.aten.linear.default` and not marked by `set_module_name_qconfig`
+    """
+    module_name_list_filters = [_get_module_name_filter(m) for m in module_name_list]
```

Comment:
I think we don't need `module_name_list` as input.

Reply:
Yes, the `filter_fn` has been revised, and the nodes whose `quantization_config` is `None` have been annotated with `_X86InductorQuantizationAnnotation(_annotated=True)`.